doc/usage/restructuredtext/directives.rst

```diff
@@ -597,7 +597,8 @@ __ https://pygments.org/docs/lexers
    .. rst:directive:option:: class: class names
       :type: a list of class names separated by spaces
 
-      The class name of the graph.
+      Class names to add to the code block's HTML ``class`` attribute when
+      building as HTML.
```

Comment:
As I mentioned in #11584, it's still not clear to me what this does for other build targets like LaTeX... From experimentation it seems like "nothing" is the answer, but is that true for all of them?

Reply:
cc: @jfbu re LaTeX.